### DISCUSSION TITLE
v0.27.8

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/UnblockNeteaseMusic#1474] [Announcements]
## TL; DR;
- Make a fortune quietly

## Announcement
So, this might be the last release due to the lack of maintenance, consider migrating to Spotify, we use this too.
Of course, feel free to take over the maintainership if you are interested in it :-)

## Features
- [refactor: Decrypt according to the 'e_r' parameter](https://github.com/UnblockNeteaseMusic/server/commit/11c1ed5928e9e9d7ac09e17afd8d536ab2dbda0b)

## Chore
- Update dependencies to the latest

## New Contributors
- @zzwtsy made their first contribution in https://github.com/UnblockNeteaseMusic/server/pull/1435
- @s12mmm3 made their first contribution in https://github.com/UnblockNeteaseMusic/server/pull/1454

**Full Changelog**: https://github.com/UnblockNeteaseMusic/server/compare/v0.27.7...v0.27.8

<hr /><em>This discussion was created from the release <a href='https://github.com/UnblockNeteaseMusic/server/releases/tag/v0.27.8'>v0.27.8</a>.</em>
